### PR TITLE
doc(pubsub): declare the library GA

### LINKED ISSUE #777: support pubsub in C/C++
Do you have plan to support pub/sub in C/C++? I see a lot of language but no C/C++
We want to use it as a substitute for kafka

### LINKED ISSUE #4878: Use release version in Doxygen docs
While in development we do not want the Doxygen docs to say "1.x.y". I am committing a change for them to say "(development)" we need to clean this up before the release.

### PR DESCRIPTION
**Pull request by coryan**
Fixes #4878 and fixes #777

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/5390)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,8 +22,8 @@
 
 **BREAKING CHANGES:**
 
-While the Pub/Sub library is not GA, and breaking changes are to be expected, we
-are close enough to a GA release that we think highlighting them is important.
+While this is the first GA release of the Pub/Sub library is not GA, and
+breaking changes were to be expected, we think highlighting them is important.
```
Comment (devbww) [inline CHANGELOG.md:26]: While this is the first GA release, it is not GA??

Comment (coryan) [reply CHANGELOG.md]: Fixed, PTAL.

**Review by devbww — APPROVED** [review]

**Comment by kamalaboulhosn** [comment]
Congratulations!

**Review by devjgm — APPROVED** [review]
Congrats! Exciting!!!

**Comment by coryan** [comment]
FTR, I did not merge this earlier because I did not like the performance numbers.

It turned out that I implemented the wrong throughput benchmarks.  In the process I fixed a number of small and medium sized performance problems, and some bugs with message ordering too.  Better outcome to have waited.

**Pull request merged**